homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_7/fibo.test.js

```diff
@@ -0,0 +1,15 @@
+const { fiboNumb } = require('./fibo');
+
+describe('Fibonacci', () => {
+    describe('test fibonacci', () => {
+        test('test fibonacci', () => {
+            expect(fiboNumb(10)[0]).toBe(0);
+        });
+        test('test fibonacci', () => {
+            expect(fiboNumb(10)[5]).toBe(5);
+        });
+        test('test fibonacci', () => {
+            expect(fiboNumb()[8]).toBe(26);
+        });
+    });
+});
```

Comment:
tests should have self describing names

also, to test full sequence you need to use a loop